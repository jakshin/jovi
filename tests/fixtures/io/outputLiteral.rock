Say mysterious
Say null

Say true
Say false

Say -42
Say -0
Say 0
Say 42

Say -12.34
Say -0.0
Say 0.0
Say 12.34

Say ""
Say "woo"
