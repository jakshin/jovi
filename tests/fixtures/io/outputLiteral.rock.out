mysterious
null
true
false
-42
0
0
42
-12.34
0
0
12.34

woo
